ods-setup/setup-ods-project.sh

```diff
@@ -38,19 +38,17 @@ if ! oc whoami; then
   exit 1
 fi
 
+# Create namespace
 if oc project ${NAMESPACE}; then
   echo "The project '${NAMESPACE}' already exists"
-  exit 1
```

Comment:
If the project exists I would exit and maybe use a flag like FORCE... It could avoid big mistakes

Reply:
Hmm yes. I was actually coming from the angle: "could this script be run multiple times?". It turns out that all those policies further down are actually idempotent.

Doing it that way it would be possible to eventually have a `make install` which works for install and for update.

But yea ... I know what you mean. Thoughts?

Reply:
e.g... I type instead of setup-ods-project.sh -n **cd**, setup-ods-project.sh -n **cds** (just a stupid typo) and imagine cds could be a project with a lot of things... tailoring it would create a huge mess

Reply:
Hmm, so the actual issue seems to be if I use it for a namespace other than `cd`. What I'm trying to add here is a Makefile target which does not allow you to choose a specific name anyway. 

So - do we need to allow other namespaces at all? I would vote to remove this feature.

Reply:
We could then put just a BIG warning saying do not run the scripts without make, or do it in your own risk :) LOL... that would be fine.